src/components/common/TmBalance.vue

```diff
@@ -346,6 +346,9 @@ export default {
     },
     totalRewardsPerDenom() {
       if (this.rewards && this.rewards.length > 0) {
+        if (!this.rewardsSentToGA) {
+          this.sendRewards(this.totalRewards)
```

Comment:
total rewards is not available yet at this point, is it?

Reply:
@Bitcoinera

Reply:
yes, I checked. But I can add it to the check for more security

Reply:
No, I mean `totalRewards` depends on `totalRewardsPerDenom` so at this point the value hasn't updated yet. As I recommended before: Create a `watch` for `totalRewards` and send it there. But we will also change this to send all the analytics inside the API.

Reply:
xDDD you suggested both things as possibilities XD

And nope totalRewards doesn't depend on totalRewardsPerDenom. Look at the code

Reply:
with this it would https://github.com/luniehq/lunie/pull/3818#discussion_r401627601

Reply:
I pushed a proposal

Reply:
What's missing to merge here?